Sources/ScipioKit/Runner.swift

```diff
@@ -219,6 +219,7 @@ extension Runner {
             case disabled
             case project
             case storage(any CacheStorage, Set<CacheActorKind>)
```

Comment:
How about introducing a new struct to hold `any CacheStorage` and `Set<CacheActorKind>`?

```swift
struct CachePolicy {
    let cacheStorage: any CacheStorage
    let actorKinds: Set<CacheActorKind>
}
```

Reply:
Yeah I agree, handling tuples is not a good experience 👍